### PR TITLE
feat: add is_idle dispositions

### PR DESCRIPTION
**Pull request by dyphire**
This is helpful for controls visibility in mpv [pseudo gui mode](https://mpv.io/manual/master/#pseudo-gui-mode).

### CODE REVIEW

File: script-opts/uosc.conf [hunk 1]
```diff
@@ -51,6 +51,7 @@ timeline_chapters_opacity=0.8
 # Item visibility control:
 #   `<[!]{disposition1}[,[!]{dispositionN}]>` - optional prefix to control element's visibility
 #   - `{disposition}` can be one of:
+#     - `idle` - true for if mpv is idle state
```
Comment (tomasklaen) [inline script-opts/uosc.conf:54]: Change to: "true if mpv is in idle mode (no file loaded)"

File: scripts/uosc.lua [hunk 2]
```diff
@@ -4280,7 +4281,11 @@ mp.observe_property('playlist-count', 'number', function(_, value)
 end)
 mp.observe_property('fullscreen', 'bool', create_state_setter('fullscreen', update_fullormaxed))
 mp.observe_property('window-maximized', 'bool', create_state_setter('maximized', update_fullormaxed))
-mp.observe_property('idle-active', 'bool', create_state_setter('idle'))
+mp.observe_property('idle-active', 'bool', function(_, idle)
+	set_state('is_idle', idle)
+	create_state_setter('idle')
```
Comment (tomasklaen) [inline scripts/uosc.lua:4286]: Remove the `create_state_setter('idle')` call. It doesn't do anything here.

**Review by tomasklaen — CHANGES_REQUESTED** [review]

File: scripts/uosc.lua (hunk 2 above)
Comment (christoph-heinrich) [reply scripts/uosc.lua:4286]: Could simply replace `'idle'` with `'is_idle'` in the original code and that should be fine or not?

Comment (tomasklaen) [reply scripts/uosc.lua:4286]: Depends on the order in which these events are being dispatched. If `idle-active` is last after everything else that triggers `dispositions` than this won't be reliable. I guess better be safe.

Comment (christoph-heinrich) [reply scripts/uosc.lua:4286]: Oh right, I missed the disposition because I only looked at that excerpt.

Comment (dyphire) [reply scripts/uosc.lua]: > Remove the `create_state_setter('idle')` call. It doesn't do anything here.

Done.

**Pull request merged**